pkgs/tools/cd-dvd/cuetools/default.nix

```diff
@@ -1,22 +1,27 @@
-{stdenv, fetchurl, automake, autoconf, flex, bison }:
+{ stdenv, fetchFromGitHub, autoreconfHook
+, bison, flac, flex, id3v2, vorbisTools
+}:
 
-stdenv.mkDerivation {
-  name = "cuetools-1.3.1";
+stdenv.mkDerivation rec {
+  name = "cuetools-${version}";
+  version = "2014-11-30"; # Uses a post 1.4.0 commit that includes an automake fix
 
-  src = fetchurl {
-    url = https://github.com/svend/cuetools/archive/cuetools-1.3.1.tar.gz;
-    sha256 = "1cap3wl0mlcqrjywpz46003w8jws05rr3r87pzqkz1g89v9459dg";
+  src = fetchFromGitHub {
+    owner = "svend";
+    repo = "cuetools";
+    rev = "365e379483ea180bcfa272d89a2524da55b1d600";
+    sha256 = "02ksv1ahf1v4cr2xbclsfv5x17m9ivzbssb5r8xjm97yh8a7spa3";
   };
 
-  preConfigure = "autoreconf -fiv";
+  nativeBuildInputs = [ autoreconfHook ];
 
-  buildInputs = [ automake autoconf flex bison ]; 
+  buildInputs = [ bison flac flex id3v2 vorbisTools ];
 
-  meta = {
-    description = "cd/dvd cue and toc file parsers and utilities";
+  meta = with stdenv.lib; {
+    description = "cuetools is a set of utilities for working with cue files and toc files";
```

Comment:
Don't start the description with repeating the package name and start of with an uppercase, no dot at the end though.